### DISCUSSION TITLE
React Toastify Transition on Update

### DISCUSSION THREAD

**Discussion by jshap2000** [fkhadra/react-toastify#806] [General]
Hi, I notice when I update a toast with the same transition, no new animation happens. Could we possibly allow for the same transition to be applied multiple times in a row to an updated toast?

**Comment by jshap2000** [comment]
Also, just wanted to add that when you call the update function on a toast, autoclose no longer works.

**Comment by tzztson** [comment]
I think that is impossible using Toastify notification.

I tried several times, but I got the same result.
